packages/shared/components/ProfileActions.svelte

```diff
@@ -81,11 +81,25 @@
     }
 
     function handleBackupClick(): void {
-        openPopup({
-            type: 'backup',
-            props: {
-                lastBackupDate,
-                lastBackupDateFormatted,
+        api.getStrongholdStatus({
```

Comment:
Without this code it's working anyway, I think it's implemented on another place at desktop code. Unless I missing something? 😅 please restore this component to the previous state. Also this behavior it's the same on settings menu.

Reply:
The idea was to hide the password field, when stronghold is unlocked but since stronghold needs a password in any case to export a backup this part is indeed unnecessary. I removed it 👍  

The stronghold image is also displayed again. Let me know if we should use it for the time being or change it to the figma illustration as discussed in the sync.